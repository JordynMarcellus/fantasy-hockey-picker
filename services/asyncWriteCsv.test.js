const fs = require("fs");
const { asyncParseCsv } = require("./asyncWriteCsv");
const mockData = {
  name: "Connor McDavid",
  position: "C",
  team: "EDM",
  rank: 1,
  rating: 89.34,
};
jest.mock("fs");

describe("asyncWriteCsv", () => {
  it("works as expected", async () => {
    const result = await asyncParseCsv([mockData]);
    expect(result).toMatchSnapshot();
  });
});
