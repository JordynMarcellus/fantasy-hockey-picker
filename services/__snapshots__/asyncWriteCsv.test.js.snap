// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`asyncWriteCsv works as expected 1`] = `
"\\"name\\",\\"position\\",\\"team\\",\\"rank\\",\\"rating\\"
\\"Connor McDavid\\",\\"C\\",\\"EDM\\",1,89.34"
`;
